py/torch_tensorrt/dynamo/runtime/_TorchTensorRTModule.py

```diff
@@ -235,8 +236,17 @@ def encode_metadata(self, metadata: Any) -> str:
     def decode_metadata(encoded_metadata: bytes) -> Any:
         dumped_metadata = base64.b64decode(encoded_metadata.encode("utf-8"))
         metadata = pickle.loads(dumped_metadata)
+        metadata["settings"].torch_executed_ops = TorchTensorRTModule.deserialize_aten_ops(metadata["settings"].torch_executed_ops)
         return metadata
 
+    @staticmethod
+    def serialize_aten_ops(aten_ops: Set[torch._ops.OpOverload]) -> Set[str]:
+        return {str(op) for op in aten_ops}
+
+    @staticmethod
+    def deserialize_aten_ops(aten_ops: Set[str]) -> Set[torch._ops.OpOverload]:
+        return {eval("torch.ops." + str(v)) for v in aten_ops}
```

Comment:
Dont use eval. You can use getattr to pull from torch ops. we do this in the autogenerated plugins if you need an example.